src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -1865,19 +1904,68 @@ const bool Matcher::match_rule_supported_vector(int opcode, int vlen, BasicType
 }
 
 const bool Matcher::match_rule_supported_vector_masked(int opcode, int vlen, BasicType bt) {
-  return false;
+  if (!UseRVV) {
+    return false;
+  }
+  switch (opcode) {
+    case Op_AddVB:
+    case Op_AddVS:
+    case Op_AddVI:
+    case Op_AddVL:
+    case Op_AddVF:
+    case Op_AddVD:
+    case Op_SubVB:
+    case Op_SubVS:
+    case Op_SubVI:
+    case Op_SubVL:
+    case Op_SubVF:
+    case Op_SubVD:
+    case Op_MulVB:
+    case Op_MulVS:
+    case Op_MulVI:
+    case Op_MulVL:
+    case Op_MulVF:
+    case Op_MulVD:
+    case Op_DivVF:
+    case Op_DivVD:
+    case Op_VectorLoadMask:
+    case Op_VectorMaskCmp:
+    case Op_AndVMask:
+    case Op_XorVMask:
+    case Op_OrVMask:
+    case Op_LoadVector:
+        opcode = Op_LoadVectorMasked;
```

Comment:
There is no break before `Op_LoadVector`, then all opcode matches before will be assigned to `Op_LoadVectorMasked`.
Is this as expected?